### DISCUSSION TITLE
Can it be deployed to serverless?

### DISCUSSION THREAD

**Discussion by yumcc-com** [medusajs/medusa#1618] [Q&A]
Can it be deployed to serverless?
For example, on aws Lambda?

**Comment by Sal-ix** [comment]
Found this:
https://medium.com/medusa-commerce/get-live-with-medusa-in-the-cloud-using-aws-4e5e8e191686

**Comment by rkreddydl** [comment]
Hi @yumcc-com 

> Can it be deployed to serverless? For example, on aws Lambda?

Were you able to deploy this on was lambda

**Reply by rkreddydl** [reply]
@yumcc-com

**Comment by sladg** [comment]
Heya! So there are some parts to this.
Backend on it's own is not deployable to serverless, because it's architecture (some pubsubs, redis, etc.) see: https://docs.medusajs.com/deployments/server

However, both Admin and Storefront (depending how you write it) are deployable to Lambdas.
I'm using [SST](http://sst.dev) with `StaticSite` and `NextJsSite` constructs.

**Reply by bruno-espino** [reply]
Hey! I know comment is kind of old, but I'm curious if you kept working with this setup and are open to have a talk about it :)

**Reply by sladg** [reply]
@bruno-espino as far as I know, the setup stayed this way :) feel free to ping me on Discord, my handle is `sladg`
